src/components/layout/ProductVersion/ProductVersion.tsx

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+import React, { CSSProperties } from 'react';
+import colors from 'cloudify-ui-common/styles/_colors.scss';
+
+export interface ProductVersionProps {
+    /**
+     * Full Cloudify version string
+     */
+    version: string;
+    /**
+     * Optional CSS style
+     */
+    style?: CSSProperties;
+}
+
+/**
+ * Formats and displays short version label (major.minor) based on passed full Cloudify version string.
+ */
+export default function ProductVersion({ version, style }: ProductVersionProps) {
+    const versionMatches = version.match(/^(\d+)\.(\d+).*$/);
+    const major = !!versionMatches && _.size(versionMatches) >= 2 ? versionMatches[1] : '';
+    const minor = !!versionMatches && _.size(versionMatches) >= 3 ? versionMatches[2] : '';
+    const shortVersion = `${major}${minor ? `.${minor}` : ''}`;
```

Comment:
Optional:

Probably we can optimize it a bit to have something like:
```suggestion
    const shortVersion = React.memo(() => {
        const versionMatches = version.match(/^(\d+)\.(\d+).*$/);
        const major = !!versionMatches && _.size(versionMatches) >= 2 ? versionMatches[1] : '';
        const minor = !!versionMatches && _.size(versionMatches) >= 3 ? versionMatches[2] : '';
        return `${major}${minor ? `.${minor}` : ''}`;
    }, [version]);
```
NOTE: I didn't check that code. It's more like to show the intention to use `React.memo`.

Reply:
Done